pkg/controller/replicaset/metrics/metrics.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"k8s.io/component-base/metrics"
+)
+
+const ReplicaSetControllerSubsystem = "replicaset_controller"
+
+var SortingDeletionAgeRatio = metrics.NewHistogram(
+	&metrics.HistogramOpts{
+		Subsystem: ReplicaSetControllerSubsystem,
+		Name:      "sorting_deletion_age_ratio",
```

Comment:
@damemi would be good to update the KEP with this change.

Reply:
Opened here: https://github.com/kubernetes/enhancements/pull/2805